lib/routes/appwrite/blog.ts

```diff
@@ -0,0 +1,89 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import logger from '@/utils/logger';
+import puppeteer from '@/utils/puppeteer';
+
+export const route: Route = {
+    path: '/blog',
+    categories: ['blog', 'reading'],
+    example: '/blog',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['appwrite.io/blog'],
+        },
+    ],
+    name: 'Appwrite Blog',
+    maintainers: ['Island'],
```

Comment:
I don't https://github.com/Island is your GitHub ID. If it does, please use that account to leave a comment on this PR.